src/internal/vendor/react-keyed-flatten-children/index.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Vendored from react-keyed-flatten-children@3.2.0 (MIT License)
```

Comment:
We need to mention the library in 3rd-party licenses, too. Example: https://github.com/cloudscape-design/test-utils/pull/81